docs/cookbook/recipe_customizing_a_mosaic_list.rst

```diff
@@ -10,11 +10,25 @@ Since version 3.0, the AdminBundle now include a mosaic list mode in order to ha
 
 It is possible to configure the default view by creating a dedicated template.
 
+.. note::
+
+    If you simply want to change the default mosaic background globally,
+    use the following configuration:
```

Comment:
If you want to change the default mosaic background globally,
please use the following configuration: